### ISSUE TITLE
[Bug] Mod with min version set to 1.8.0 getting incompatible warning

### ISSUE THREAD

**Issue by linuxgurugamer** [KSP-CKAN/CKAN#3004]
This happens on a lot of mods, but here is one case:

AnyRes, the KSP_VERSION_MIn is 1.8.0.
CKAN allows it to be installed

When launching game, I get a warning:
![image](https://user-images.githubusercontent.com/1630166/74490055-561c2480-4e95-11ea-9f78-ec9332f9c262.png)

**Comment by HebaruSan** [comment]
No, it's not just the min that's set; this module is compatible **only** with 1.8.0:

https://github.com/KSP-CKAN/CKAN-meta/blob/ca0103d6ad2040cbc7c93bcc904892d32b8f176f/AnyResContinued/AnyResContinued-2.0.3.3.ckan#L8

```json
    "ksp_version": "1.8.0",
```

Presumably this is yet another repeat of KSP-CKAN/NetKAN#7166, the fact that SpaceDock doesn't do unbounded ranges. Solution is the same:

> If you want an open-ended version range defined completely by the AVC file, then a host that requires selection of the compatible game version is not for you. GitHub is a better fit.
